docs/product-installation-guide/src/main/asciidoc/run-standalone-properties-con.adoc

```diff
@@ -1,5 +1,12 @@
-[id='install-standalone-properties-con']
+[id='run-standalone-properties-con']
 = Supported properties
+When you install standalone {CENTRAL}, you can use the properties listed in this section in the following command:
+[source] 
+----
+rhdm-7.0.0.GA-decision-central-standalone.jar -s application-config.yaml -D<property>=<value> -D<property>=<value>
```

Comment:
Sorry, I missed this in the previous jar: you're missing java -jar in the command to run the standalone jar, so the whole command should be java -jar rhdm-7.0.0.GA-decision-central-standalone.jar -s application-config.yaml -D<property>=<value> -D<property>=<value>